README.md

```diff
@@ -45,6 +45,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <td align="center"><a href="https://github.com/miguelsmuller"><img src="https://avatars.githubusercontent.com/u/4589909?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Miguel Müller</b></sub></a><br /><a href="#translation-miguelsmuller" title="Translation">🌍</a></td>
     <td align="center"><a href="https://atzu.ml"><img src="https://avatars.githubusercontent.com/u/32654584?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Azurnex</b></sub></a><br /><a href="#translation-z3ro0k" title="Translation">🌍</a></td>
     <td align="center"><a href="http://snazzah.com"><img src="https://avatars.githubusercontent.com/u/7025343?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Snazzah</b></sub></a><br /><a href="https://github.com/uetchy/namae/commits?author=Snazzah" title="Code">💻</a></td>
+    <td align="center"><a href="http://thebigbot.mod.land"><img src="https://avatars.githubusercontent.com/u/77632836?s=48&amp;v=4" width="100px;" alt=""/><br /><sub><b>Thebigbot</b></sub></a><br /><a href="https://github.com/uetchy/namae/commits?author=Thebigbot0000" title="Code">🌍</a></td>
```

Comment:
You don't have to do this yourself. We'll add you here anyways after your PR gets merged.

Reply:
By tonight I send it to you without that piece of code (European time)